system-addon/test/schemas/pings.js

```diff
@@ -63,14 +63,34 @@ const SessionPing = Joi.object().keys(Object.assign({}, baseKeys, {
   session_id: baseKeys.session_id.required(),
   page: baseKeys.page.required(),
   session_duration: Joi.number().integer().required(),
-  action: Joi.valid("activity_stream_session").required()
+  action: Joi.valid("activity_stream_session").required(),
+  perf: Joi.object().keys({
+    // Timestamp of the action perceived by the user to trigger the load
+    // of this page.
+    //
+    // Not required at least for error cases where the
+    // observer event doesn't fire
+    load_trigger_ts: Joi.number().positive(),
```

Comment:
Leaving the .required() off after discussion.  I've updated the schema in pings.js and the data dictionary with indicators that we want this tracked on the server side.